bundles/org.openhab.binding.yamahamusiccast/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,20 @@
 	<description>This is the binding for Yamaha Musiccast</description>
 	<connection>local</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>upnp</service-type>
+			<match-properties>
+				<match-property>
+					<name>manufacturer</name>
+					<regex>(?i).*Yamaha.*</regex>
```

Comment:
As I understand, `(?i)` turns on case-insensitive mode. So if you keep it, you will catch value containing either Yamaha or YAMAHA.
Exactly the same result as in the "Yamaha Receiver" binding.

Reply:
True, I will fix that.